drivers/sensor/max1726x/max1726x.c

```diff
@@ -74,6 +74,26 @@ static void convert_millis(struct sensor_value *val, int32_t val_millis)
 	val->val2 = (val_millis % 1000) * 1000;
 }
 
+/**
+ * @brief Set hibernate mode
+ *
+ * @param dev MAX1726X device to access
+ * @return 0 if successful, or negative error code
+ */
+static int max1726x_set_hibernate(const struct device *dev)
```

Comment:
I'm not familiar with this sensor, I wonder if hibernate mode can be related with device power management.

Reply:
Hum, i am not really sure : 
- On the one hand I think it is
- On the other hand, i want to use this function (same for the shutdown function) from external zephyr code to force the max1726x sensor power state (and switch between active, hibernate and shutdown)

Reply:
Is it because current device PM is failing to suit your needs? If so, can you explain the reasons? Some work is ongoing to make device PM better, so any feedback will be beneficial.